packages/react/src/components/InlineLoading/InlineLoading.tsx

```diff
@@ -121,33 +129,34 @@ const InlineLoading = ({
         />
       );
     }
-    if (status === 'inactive') {
+    else if (status === 'inactive') {
       if (!iconDescription) {
         iconLabel = 'not loading';
       }
-      return (
-        <title className={`${prefix}--inline-loading__inactive-status`}>
-          {iconLabel}
-        </title>
+      loading.animation = (
+        <svg width="0" height="0" viewBox="0 0 0 0">
+          <title className={`${prefix}--inline-loading__inactive-status`}>
+            {iconLabel}
+          </title>
+        </svg>
```

Comment:
Is it correct to use a custom svg here that's effectively blank or a placeholder? Are there any accessibility implications related to the latter?